extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -212,14 +213,21 @@ export class InstrumentationService {
    * @private
    */
   createVisibilityListener_(callback, config) {
-    if (config['visibilitySpec']) {
+    const spec = config['visibilitySpec'];
+    const selector = config['selector'] || (spec && spec['selector']);
+    const analyticsVariables = config[vars] || {};
```

Comment:
i don't understand this line. why are we reading from config[vars]? is some value expected to be there?